app/lib/features/events/providers/event_providers.dart

```diff
@@ -1,225 +1,198 @@
-import 'dart:async';
-
-import 'package:acter/features/home/providers/client_providers.dart';
-import 'package:acter_flutter_sdk/acter_flutter_sdk.dart';
+import 'package:acter/features/events/event_utils/event_utils.dart';
+import 'package:acter/features/events/providers/notifiers/event_notifiers.dart';
 import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' as ffi;
 import 'package:riverpod/riverpod.dart';
 
-final spaceEventsProvider = AsyncNotifierProvider.family<
```

Comment:
so we are essentially moving from ```.subscribeStream()``` methods which were more streamlined with client updates?